src/screens/CreatePost/elements/SheetExpiredPost.js

```diff
@@ -4,54 +4,53 @@ import {StyleSheet, Text, View} from 'react-native';
 import RBSheet from 'react-native-raw-bottom-sheet';
 
 import FlatListItem from '../../../components/FlatListItem';
+import {colors} from '../../../utils/colors';
 import {fonts} from '../../../utils/fonts';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been updated to use lowercase `colors` and `fonts` instead of the previous uppercase `COLORS`. This change seems fine as long as the exported constants in the respective modules are also updated accordingly. If not, this will cause a reference error.
```diff
- import {COLORS} from '../../../utils/theme';
+ import {colors} from '../../../utils/colors';
  import {fonts} from '../../../utils/fonts';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->